### PR TITLE
Enable logging to seq from both .net framework and .net standard/core applications

### PR DESCRIPTION
**Pull request by developerlens**
Enable logging to seq from both .net framework and .net standard/core applications
Added a .net core application to reflect implementation in .net core 2.x
Updated readme to reflect chanages made

### CODE REVIEW

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj [hunk 1]
```diff
@@ -1,116 +1,23 @@
...
   <ItemGroup>
-    <None Include="..\..\asset\Seq.Client-Public.snk">
-      <Link>Seq.Client-Public.snk</Link>
-    </None>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-    <None Include="Seq.Client.Log4Net.nuspec">
-      <SubType>Designer</SubType>
-    </None>
+    <None Include="..\..\asset\Seq.Client-Public.snk" Link="Seq.Client-Public.snk" />
```
Comment (nblumhardt) [inline src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: This won't be enough to cause the assembly to be signed; needs `<SignAssembly>` and `<PublicSign>`, e.g. https://github.com/datalust/nlog-targets-seq/blob/dev/src/NLog.Targets.Seq/NLog.Targets.Seq.csproj#L11

**Comment by developerlens** [comment]
I have update the project to include all suggestion from the comments. Thanks

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
-# Seq.Client.Log4Net [![Build status](https://ci.appveyor.com/api/projects/status/sxw4n1a6v9o7db2i?svg=true)](https://ci.appveyor.com/project/datalust/seq-client)
+# SEQ Implementation for log4net [![Build status](https://ci.appveyor.com/api/projects/status/sxw4n1a6v9o7db2i?svg=true)](https://ci.appveyor.com/project/datalust/seq-client)
```
[the remark this answers is not in the document]
Comment (nblumhardt) [reply README.md]: Looks like the revert on README.md was missed

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj [hunk 3]
```diff
@@ -1,116 +1,23 @@
...
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{CBDC0A30-5EC1-4DFE-AB0A-8CFC181B7DAB}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Seq</RootNamespace>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>1.0</Version>
```
[the remark this answers is not in the document]
Comment (nblumhardt) [reply src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: Looks like this one was missed.

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj [hunk 4]
```diff
@@ -1,116 +1,23 @@
...
+    <PackageTags>seq log4net structured logging events</PackageTags>
+    <Product />
+    <Copyright>http://apache.org/licenses/LICENSE-2.0.html</Copyright>
     <AssemblyName>Seq.Client.Log4Net</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-    <TargetFrameworkProfile />
+    <RootNamespace>Seq.Client.Log4Net</RootNamespace>
```
[the remark this answers is not in the document]
Comment (nblumhardt) [reply src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: Looks like this one was missed.

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj [hunk 5]
```diff
@@ -1,116 +1,30 @@
...
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <DocumentationFile>bin\Release\Seq.Client.Log4Net.XML</DocumentationFile>
-    <LangVersion>6</LangVersion>
-  </PropertyGroup>
-  <PropertyGroup>
+    <RootNamespace>Seq.Client.Log4Net</RootNamespace>
     <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageTags>seq;log4net</PackageTags>
+    <PackageProjectUrl>https://github.com/datalust/seq-client-log4net</PackageProjectUrl>
```
Comment (nblumhardt) [inline src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: This is a duplicate, there's still an incorrect `<PackageProjectUrl>` a few lines up :-).

**Comment by nblumhardt** [comment]
Just a quick note in case you haven't spotted it - the ["Files" tab](https://github.com/datalust/seq-client-log4net/pull/40/files) shows comments in the context of changes and earlier comments - it's usually much easier to keep track of PR state there, than in the default ("Conversation") view in GitHub. HTH!

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj (hunk 4 above)
[the remark this answers is not in the document]
Comment (developerlens) [reply src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: Added it

File: src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj (hunk 5 above)
Comment (developerlens) [reply src/Seq.Client.Log4Net/Seq.Client.Log4Net.csproj]: Removed the duplicate

**Comment by developerlens** [comment]
Made All Changes and pushed back

**Comment by nblumhardt** [comment]
Awesome! Thanks, and much appreciated.

I've set up a `dev` branch for this to target, so that we can iron out any packaging issues etc. before stamping this "RTM" by merging from there into `master`.

Thanks again!

**Pull request merged**